path-following.ts

```diff
@@ -84,6 +84,26 @@ namespace scene {
     //% path.defl="locationTiles"
     //% group="Tiles" weight=9
     export function followPath(sprite: Sprite, path: tiles.Location[], speed?: number) {
+        const currentLocation = locationOfSprite(sprite)
```

Comment:
Oh, also can we add null checks on sprite & path here to gracefully handle the 'no path' case? e.g. do nothing if !path